docs/source/advanced/multi_gpu.rst

```diff
@@ -639,38 +639,39 @@ The reason is that the full batch is visible to all GPUs on the node when using
 
 ----------
 
-TorchElastic
---------------
-Lightning supports the use of TorchElastic to enable fault-tolerant and elastic distributed job scheduling. To use it, specify the 'ddp' or 'ddp2' backend and the number of gpus you want to use in the trainer.
+Torch Distributed Elastic
+-------------------------
+Lightning supports the use of Torch Distributed Elastic to enable fault-tolerant and elastic distributed job scheduling. To use it, specify the 'ddp' or 'ddp2' backend and the number of gpus you want to use in the trainer.
 
 .. code-block:: python
 
     Trainer(gpus=8, accelerator='ddp')
 
-
-Following the `TorchElastic Quickstart documentation <https://pytorch.org/elastic/latest/quickstart.html>`_, you then need to start a single-node etcd server on one of the hosts:
+To launch a fault-tolerant job, run the following on all nodes.
```

Comment:
@edenafek - do you think its worth having a dedicated docs section on which schedulers/launchers Lightning supports? This could cover launching with SLURM, torch distributed elastic, etc